README.md

```diff
@@ -6,18 +6,21 @@ Prisma [templates](https://github.com/prisma/prisma-schema-examples) packaged up
 
 [Documentation](https://paka.dev/npm/@prisma/templates)
 
-## Start developing:
+## Development:
 
 
-1) Install sha1sum library
-2) `yarn run build:gen`
-3) if `yarn test` fails - run `yarn test -u`
+1. Install `sha1sum` library (for macOS via `brew install md5sha1sum`).
```

Comment:
I didn't have `sha1sum` on my machine, but the generate script did not throw an error. It just silently kept returning the same cache ID. I improved things by at least specifying the install command to use (which was also non-obvious).

In the future we should look at a task running like turborepo or Nx which have solved these kinds of cache issues in their tools.